handshake-messages.go

```diff
@@ -90,10 +90,16 @@ func (ch ClientHelloBody) Truncated() ([]byte, error) {
 		return nil, fmt.Errorf("tls.clienthello.truncate: Last extension is not PSK")
 	}
 
-	chm, err := HandshakeMessageFromBody(&ch)
+	// TODO(ekr@rtfm.com): This will not work right with DTLS.
```

Comment:
Why not?

Reply:
I'm trying to remember TBH. I'll remove it and I guess we'll find out :)